Header header
string learner    # Supported: '2D' and '3D'.
                  # Identifies wether the object_id belongs to the 2D or the 3D template.
int32 object_id   # ID of the template for this learner.
